ui-v2/eslint.config.js

```diff
@@ -7,13 +7,15 @@ import globals from "globals";
 import tseslint from "typescript-eslint";
 import testingLibrary from "eslint-plugin-testing-library";
 import jestDom from "eslint-plugin-jest-dom";
+import pluginQuery from "@tanstack/eslint-plugin-query";
 
 export default tseslint.config(
 	{ ignores: ["dist", "src/api/prefect.ts"] },
 	{
 		extends: [
 			js.configs.recommended,
 			...tseslint.configs.recommendedTypeChecked,
+			...pluginQuery.configs["flat/recommended"],
```

Comment:
If we're using the `flat/recommend` config, then I think this goes right above the `...pluginRouter.configs["flat/recommended"]` entry ([link to docs](https://tanstack.com/query/latest/docs/eslint/eslint-plugin-query#recommended-setup)).

Reply:
Resolved in latest pushed commit: https://github.com/PrefectHQ/prefect/pull/16155/commits/16ac123e7c5c19ee0a33a3b206e6ef1cd2e98748